documentation/modules/ROOT/pages/transformations/filtering.adoc

```diff
@@ -141,15 +141,12 @@ The following table lists the variables that {prodname} binds into the evaluatio
 |`keySchema` |Schema of the message key.|`org.apache.kafka.connect{zwsp}.data{zwsp}.Schema`
 |`valueSchema`|Schema of the message value.| `org.apache.kafka.connect{zwsp}.data{zwsp}.Schema`
 |`topic`|Name of the target topic.| String
-|`headers`
-a|A Java map of message headers. The key field is the header name.
+|`header`|A Java map of message headers. The key field is the header name.
```

Comment:
IMHO the name `headers` was better but changing it now would break existing filter expressions. 😞

Reply:
https://github.com/debezium/debezium/blob/v2.6.1.Final/debezium-scripting/debezium-scripting/src/main/java/io/debezium/transforms/scripting/Jsr223Engine.java#L72